src/Metrics.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Utopia;
+
+use Utopia\Telemetry\Adapter as Telemetry;
+use Utopia\Telemetry\Histogram;
+use Utopia\Telemetry\UpDownCounter;
+
+class Metrics
+{
+    private Histogram $requestDuration;
+    private UpDownCounter $activeRequests;
+    private Histogram $requestBodySize;
+    private Histogram $responseBodySize;
+
+    public function __construct(Telemetry $telemetry)
```

Comment:
Note sure about this structure, it will be very hard to enforce it in the future as this are just basically helper methods. Can we just make this protected methods in the App class or even just add the code inline?